site/_posts/2021-05-11-manageiq-perf-results.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "ManageIQ performance results and improvements"
+author: Thad Jennings
+date: 2021-05-11
+tags: performance
+comments: true
+published: true

```

Comment:
These were marked for no comments but I thought it would be good to mark it as published and accept comments.